codespell_lib/data/dictionary.txt

```diff
@@ -10572,6 +10589,7 @@ calcalation->calculation
 calcalations->calculations
 calcalator->calculator
 calcalators->calculators
+calcaulte->calculate
```

Comment:
It would be great if you could add all verbal forms:
```suggestion
calcaulte->calculate
calcaulted->calculated
calcaultes->calculates
calcaulting->calculating
```